_posts/2022-04-04-grant-announcements.md

```diff
@@ -0,0 +1,182 @@
+
+[@palazzovincenzo][vincenzo twitter]
+
+## Niklas Gögge
+
+<img src="/assets/images/niklas-goegge.jpg" alt="Picture of Niklas Gögge" style="float:left;border:1px solid
+darkgray;margin-right:25px;width:150px;" />
+
+Niklas writes of his Bitcoin experience and goals:
+
+I first got interested in Bitcoin in 2017, but it wasn’t until late 2018 when I
+read Mastering Bitcoin that I got fully obsessed with the technical side of Bitcoin.
+
+With the start of the pandemic I had a lot of time on my hands and
+start contributing to Bitcoin. I found [multiple talks][tadge talks] of Tadge Dryja talking
+about Utreexo which prompted me to hop on IRC and ask where I can help out. The
+[Utreexo][utreexo website] project introduces a hash based dynamic accumulator for Bitcoin’s UTXO
+set. A dynamic accumulator is a compact representation of a set to which
+elements can be added, removed, and proven. Utreexo aims to make it easier to run
+full nodes by trading off the storage requirements of the UTXO set for a
+bandwidth increase.
+
+In 2021 I worked at Mozilla as a part time student worker on the Firefox
+security infrastructure team. In my spare time, while working at Mozilla and
+finishing up my bachelors degree, I ported the Utreexo accumulator library to
+C++ ([libutreexo][libutreexo github]) and worked on a prototype Utreexo implementation for Bitcoin
+Core. I participated in the [PR review club][pr review club] and opened some of my own PRs
+([#22340][bitcoin 22340], [#21603][bitcoin 21603]).
+
+The grant from Brink gives me the opportunity to work full time on Bitcoin Core
+without distractions. I will continue my work on the C++ implementation of
+Utreexo and propose a proof-of-concept PR that people can test and review. In
+addition to working on Utreexo I plan on making more security focused
+contributions to Bitcoin Core and review PRs.
+
+[@dergoegge][niklas twitter]
+
+## Bruno Garcia
+
+<img src="/assets/images/bruno-garcia.jpg" alt="Picture of Bruno Garcia" style="float:left;border:1px solid
+darkgray;margin-right:25px;width:150px;" />
+
+Bruno writes of his Bitcoin experience and goals:
+
+I discovered Bitcoin in 2015 and fell in love with the concept of digital money
+and how this could improve and collaborate for freedom. Years later, I started
+studying the Bitcoin protocol more, reading books like Programming Bitcoin,
+Mastering Bitcoin, and other materials from the internet.
+
+In 2021, I attended the Chaincode [Bitcoin Protocol Development
+Seminar][chaincode learning] where I learned more about the
+Bitcoin protocol and started contributing to Bitcoin Core. Since I have a DevSec
+and QA background, my journey started by reviewing and reading the [functional
+tests][bitcoin functional tests]. This is how I discovered an
+inconsistency in the taproot test and got my first PR merged ([#21081][bitcoin 21081]).
+
+Since then, I haven’t stopped contributing to Bitcoin Core by testing and
```

Comment:
Nit: s/haven't stopped/have continued/

(In English, it's preferred to use the positive form of a phrase rather than the negative.)